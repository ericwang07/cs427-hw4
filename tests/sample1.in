4 6 4 6 5 4 4 1 5 6 q
