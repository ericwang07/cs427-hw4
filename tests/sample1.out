Welcome to Kalah!

----------
South's turn:
           N1    N2    N3    N4    N5    N6
          [ 4]  [ 4]  [ 4]  [ 4]  [ 4]  [ 4]
  N0 [ 0]                                    [ 0] S0
          [ 4]  [ 4]  [ 4]  [ 4]  [ 4]  [ 4]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for South (q to quit): 
South playing move 4
South gets another turn

----------
South's turn:
           N1    N2    N3    N4    N5    N6
          [ 4]  [ 4]  [ 4]  [ 4]  [ 4]  [ 4]
  N0 [ 0]                                    [ 1] S0
          [ 4]  [ 4]  [ 0]  [ 5]  [ 5]  [ 5]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for South (q to quit): 
South playing move 6
Turn is over

----------
North's turn:
           N1    N2    N3    N4    N5    N6
          [ 4]  [ 4]  [ 4]  [ 4]  [ 4]  [ 4]
  N0 [ 0]                                    [ 1] S0
          [ 0]  [ 5]  [ 1]  [ 6]  [ 6]  [ 5]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for North (q to quit): 
North playing move 4
North gets another turn

----------
North's turn:
           N1    N2    N3    N4    N5    N6
          [ 5]  [ 5]  [ 5]  [ 0]  [ 4]  [ 4]
  N0 [ 1]                                    [ 1] S0
          [ 0]  [ 5]  [ 1]  [ 6]  [ 6]  [ 5]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for North (q to quit): 
North playing move 6
Turn is over

----------
South's turn:
           N1    N2    N3    N4    N5    N6
          [ 5]  [ 6]  [ 6]  [ 1]  [ 5]  [ 0]
  N0 [ 1]                                    [ 1] S0
          [ 0]  [ 5]  [ 1]  [ 6]  [ 6]  [ 5]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for South (q to quit): 
South playing move 5
South gets another turn

----------
South's turn:
           N1    N2    N3    N4    N5    N6
          [ 5]  [ 6]  [ 6]  [ 1]  [ 5]  [ 0]
  N0 [ 1]                                    [ 2] S0
          [ 0]  [ 0]  [ 2]  [ 7]  [ 7]  [ 6]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for South (q to quit): 
South playing move 4
Turn is over

----------
North's turn:
           N1    N2    N3    N4    N5    N6
          [ 5]  [ 6]  [ 6]  [ 1]  [ 5]  [ 0]
  N0 [ 1]                                    [ 2] S0
          [ 0]  [ 0]  [ 0]  [ 8]  [ 8]  [ 6]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for North (q to quit): 
North playing move 4
Turn is over

----------
South's turn:
           N1    N2    N3    N4    N5    N6
          [ 5]  [ 6]  [ 7]  [ 0]  [ 5]  [ 0]
  N0 [ 1]                                    [ 2] S0
          [ 0]  [ 0]  [ 0]  [ 8]  [ 8]  [ 6]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for South (q to quit): 
South playing move 1
Turn is over

----------
North's turn:
           N1    N2    N3    N4    N5    N6
          [ 5]  [ 7]  [ 8]  [ 1]  [ 6]  [ 1]
  N0 [ 1]                                    [ 3] S0
          [ 0]  [ 0]  [ 0]  [ 8]  [ 8]  [ 0]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for North (q to quit): 
North playing move 5
Turn is over

----------
South's turn:
           N1    N2    N3    N4    N5    N6
          [ 6]  [ 8]  [ 9]  [ 2]  [ 0]  [ 1]
  N0 [ 2]                                    [ 3] S0
          [ 1]  [ 0]  [ 0]  [ 8]  [ 8]  [ 0]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for South (q to quit): 
South playing move 6
Captured 9 seeds
Turn is over

----------
North's turn:
           N1    N2    N3    N4    N5    N6
          [ 6]  [ 0]  [ 9]  [ 2]  [ 0]  [ 1]
  N0 [ 2]                                    [12] S0
          [ 0]  [ 0]  [ 0]  [ 8]  [ 8]  [ 0]
           S6    S5    S4    S3    S2    S1

Please enter a pit number for North (q to quit): 
Quitting game at user's request
Game over -- goodbye!
